docs/content/configuration-language.mdx

```diff
@@ -580,6 +582,76 @@ This particular use of referencing relations on related objects is defining a tr
 
 This can be used to indicate that **viewers** of a **folders** are **viewers** of all **documents** in that **folder**.
 
+:::caution
+<details>
+<summary>
+Note that when searching tuples that are related to the object (the word after `from`, also called the tupleset), <ProductName format={ProductNameFormat.LongForm}/> will not do any evaluation and will only consider concrete objects (of the form `object_type:object_id`) that were directly assigned, and will throw an error if it encounters any rewrites, or a `*`, a type bound public access (`object_type:*`) or a userset (`object_type:object_id#relation`).
+</summary>
+  In the case below:
```

Comment:
do we want to mention that the model below is 1.0.  In the case of model 1.1, typecheck restriction will prevent you from writing such a model?